nh_eobs_mental_health/models/nh_clinical_wardboard.py

```diff
@@ -0,0 +1,91 @@
+from openerp.osv import orm, osv, fields
```

Comment:
I think we should keep them especially considering that there are `nh.clinical.api` and `nh.eobs.api` models so it would be good to know what it's overriding in this sense